src/http-context.ts

```diff
@@ -48,7 +48,7 @@ export class HttpContext {
     }
 
     const { headers, header, url: rawUrl, method: m } = req.json();
-    const method = m ?? 'GET';
+    const method = m?.toUpperCase() ?? 'GET';
```

Comment:
Huh, TIL `?.` earlier in the chain cascades to the `()`! I would've thought `m?.toUpperCase?.()` would be required.

Reply:
i think it works for methods too? at least i've been doing this in other places. the compiler didn't complain but also i think it's an `any`. This does seem to work though:

```typescript
function z(s?: string | null) {
  console.log(s?.toUpperCase() || "GET")
}

z()
```

Reply:
becomes:

```javascript
"use strict";
function z(s) {
    console.log((s === null || s === void 0 ? void 0 : s.toUpperCase()) || "GET");
}
z();
```